dali/base/dasds.cpp

```diff
@@ -80,8 +81,6 @@ static unsigned readWriteTimeout = 60000;
 #define DEFAULT_LCIDLE_RATE 1          // 1 write transactions per idle period. <= this rate is deemed idle (suitable for save)
 #define STORENOTSAVE_WARNING_PERIOD 72 // hours
 
-static unsigned msgCount=(unsigned)-1;
-
 // # HELP sds_requests The total number of Dali SDS requests handled
```

Comment:
Minor: The following comments are also redundant and should go.

Reply:
In my opinion these comments are now no longer needed. The instantiation of the metric itself has the description which should suffice. If additional comments are needed referencing the individual metrics, I can restore them.